### ISSUE TITLE
Latest version (2.0.0) not on Nuget

### ISSUE THREAD

**Issue by tony-johnson-i2o** [fsprojects/FSharp.TypeProviders.StarterPack#126]
### Description

When I download and build this locally it builds as version 2.0.0. This is different from the latest available on nuget 1.1.3.88. Is there a reason 2.0.0 hasn't been/can't be uploaded to nuget?

**Comment by sergey-tihon** [comment]
What is your use case? I think that that we just do not use NuGet anymore to distribute type-provider (TP) files.

All community TP projects use [Paket](https://fsprojects.github.io/Paket/) and Paket allow us to reference files from this repository directly and do update when version change. See [sample](https://github.com/fsprojects/SwaggerProvider/blob/master/paket.dependencies#L18-L20)

**Comment by dsyme** [comment]
@sergey-tihon We should really keep the nuget package up-to-date too I think

**Comment by tony-johnson-i2o** [comment]
Yes, we're using Paket also and we probably will take your suggested approach. As a newcomer to this project, it is not obvious that the version on Nuget (which is still where I think a lot of people will start when adding a dep) is _not_ the latest. Given that the repo readme also suggests the latest version is 1.1.3.88 (the nuget version) as well, it was only when I started to dig into the actual included `.fs` files that I realised the two were different.

To avoid similar confusion for other newcomers, I think it preferable to either remove from nuget altogether, or ensure the latest version is hosted there.
